integration/src/main/java/scripts/testscripts/DatasetOperations.java

```diff
@@ -63,7 +66,8 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
 
   private void setupWorkspace(TestUserSpecification user) throws Exception {
     rawlsClient = new RawlsClient(server, user);
-    workspaceDetails = rawlsClient.createTestWorkspace();
+    workspaceSource = rawlsClient.createTestWorkspace();
+    workspaceDest = rawlsClient.createTestWorkspace();
```

Comment:
Since this is only used inside `exportUserJourney()` I would create it there. Also when we test both TDR and workspace export we'd want to make a new destination workspace each time.